edit-post/components/sidebar/post-schedule/index.js

```diff
@@ -3,27 +3,35 @@
  */
 import { __ } from '@wordpress/i18n';
 import { PanelRow, Dropdown, Button } from '@wordpress/components';
+import { withInstanceId } from '@wordpress/compose';
 import { PostSchedule as PostScheduleForm, PostScheduleLabel, PostScheduleCheck } from '@wordpress/editor';
 
 /**
  * Internal dependencies
  */
 import './style.scss';
 
-export function PostSchedule() {
+export function PostSchedule( { instanceId } ) {
 	return (
 		<PostScheduleCheck>
 			<PanelRow className="edit-post-post-schedule">
-				<span>{ __( 'Publish' ) }</span>
+				<label
+					htmlFor={ `edit-post-post-schedule__toggle-${ instanceId }` }
```

Comment:
While technically buttons are [labelable elements](https://www.w3.org/TR/html52/sec-forms.html#labelable-element), this label also overrides the button text. The accessible name is given but this label so **the date doesn't get announced**. You can easily verify this in the Chrome dev tool, in the Accessibility tab, under Computer Properties > Name: "Publish". This issue leads also to a more general issue: buttons and interactive UI controls should announce the available action, not the underlying state. See https://github.com/WordPress/gutenberg/issues/470. I'll add more details in an extended comment.